sql/1.110/rollback/rollback.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
+
+ALTER TABLE users RENAME COLUMN github_user_plaintext TO github_user;
```

Comment:
@jbarrett This should also remove the github_id column. When I used rollback and then got back to the branch and reapplied the changes I got this error:
<pre>
BEGIN
ALTER TABLE
ERROR:  column "github_id" of relation "users" already exists
ERROR:  current transaction is aborted, commands ignored until end of transaction block
ERROR:  current transaction is aborted, commands ignored until end of transaction block
ERROR:  current transaction is aborted, commands ignored until end of transaction block
ERROR:  current transaction is aborted, commands ignored until end of transaction block
ERROR:  current transaction is aborted, commands ignored until end of transaction block
ERROR:  current transaction is aborted, commands ignored until end of transaction block
ROLLBACK
</pre>